brian2/stateupdaters/exponential_euler.py

```diff
@@ -33,7 +33,7 @@ def get_conditionally_linear_system(eqs, variables=None):
     --------
     >>> from brian2 import Equations
     >>> eqs = Equations("""
-    ... dv/dt = (-v + w**2) / tau : 1
+    ... dv/dt = (-v + w**2.0) / tau : 1
```

Comment:
Does this have performance implications?

Reply:
Given that it is just a change in the doctest, I'm quite sure it does not ;)
But in general it is the same issue as above: before `w**2` and `w**2.0` where both converted to `w**2.0` when we handed them to sympy – now they are given to sympy as either an integer or a float.

Reply:
OK!